package.json

```diff
@@ -7,22 +7,22 @@
   "scripts": {
     "eslint": "eslint packages/ --ext .ts",
     "prettier": "prettier --check './packages/**/*.ts' '!**/*.d.ts'",
-    "clean": "rimraf packages/*/dist bin",
-    "reset": "yarn clean && rimraf .parcel-cache packages/*/node_modules && rimraf node_modules",
-    "package": "yarn clean && yarn build && node ./scripts/package",
-    "release": "yarn node ./scripts/release",
+    "clean": "mkdirp packages/base/dist && rimraf packages/*/dist bin etc/jest/.test-data",
```

Comment:
You create a dist folder inside the base package and then do an `rm -rf` for every dist folder under every package. Is it because otherwise, `rimraf` would raise an error saying that there is no dist folder in the base package?

Reply:
It just says that there is no match for packages/*/dist